packages/react/dialog/src/Dialog.tsx

```diff
@@ -244,24 +249,20 @@ const DialogContentModal = React.forwardRef((props, forwardedRef) => {
 
 const DialogContentNonModal = React.forwardRef((props, forwardedRef) => {
   const context = useDialogContext(CONTENT_NAME);
-  const isPointerDownOutsideRef = React.useRef(false);
+  const isEscapeKeyPressRef = React.useRef(false);
```

Comment:
nit: Shall we name this `isEscapeKeyDown` considering that's the name of the handler?